schema-engine/core/src/lib.rs

```diff
@@ -43,7 +43,7 @@ fn connector_for_connection_string(
                 preview_features,
                 shadow_database_connection_string,
             };
-            let mut connector = SqlSchemaConnector::new_postgres();
+            let mut connector = SqlSchemaConnector::new_postgres_like();
```

Comment:
We don't know the exact provider here since we don't have a schema, it will be determined later after connecting.